src/Synth/LFO.cpp

```diff
@@ -50,7 +56,10 @@ LFO::LFO(const LFOParams &lfopars, float basefreq, const AbsTime &t, WatchManage
     else {
         const float tmp = fmod(t.time() * phaseInc, 1.0f);
         phase = fmod((lfopars.Pstartphase - 64.0f) / 127.0f + 1.0f + tmp, 1.0f);
```

Comment:
somehow the global sync doesn't seem to work anymore despite the line being executed - proven by printf